### PR TITLE
Add MongoDB Atlas SQL Interface documentation

### PR DESCRIPTION
**Pull request by bucaojit**
Add the documentation for MongoDB Atlas SQL Interface connector, as a step in the certifying process.

### CODE REVIEW

**Comment by bucaojit** [comment]
@microsoft-github-policy-service agree company="MongoDB"

**Comment by Jak-MS** [comment]
@bezhan-msft -
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.

NOTE: as a new article, this will need to go via the private repo.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by DougKlopfenstein** [comment]
Add "bezhan" as the ms.author.

File: powerquery-docs/connectors/mongodb-atlas-sql-interface.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+> [!Note]
+> The following connector article is provided by MongoDB, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the MongoDB website and use the support channels there.
+
+## Summary
+
+| Item                               | Description                  |
+|------------------------------------|------------------------------|
+| Release State                      | Beta                         |
+| Products                           | Power BI (datasets)          |
```
[the remark this answers is not in the document]
Comment (bucaojit) [reply powerquery-docs/connectors/mongodb-atlas-sql-interface.md]: Thanks for the pointers.  I updated the table to indicate that Dataflows are are supported.  And the section on how to connect using Power Query Online.  I also added images to this and the previous Power Query Desktop section.

**Review by DougKlopfenstein — APPROVED** [review]

**Comment by DougKlopfenstein** [comment]
#sign-off

**Pull request merged**